### ISSUE TITLE
Improve code readability with PEP8 Style Guide for Python Code

### ISSUE THREAD

**Issue by txenoo** [iago1460/django-radio#24]
https://www.python.org/dev/peps/pep-0008/

**Comment by iago1460** [comment]
I do not agree entirely with pep8 rules, the [max line length](https://code.djangoproject.com/ticket/23395) for this project is 120 and the way to split big function calls like: 

```python
value = function_call(
    arg1, arg2
)
```
The commit 6d92bca8528f9e99db54392c6e349bd23ed4971f should fix the majority of this issues

**Comment by stefan-walluhn** [comment]
I merged your changes you committed in 6d92bca. There are still some changes remaining in my pull request. I commented all my changes at
  https://github.com/iago1460/django-radio/pull/23/files

I did not touch the migration 0001_initial.py again, since there were no changes in 6d92bca. Let me know, if there is something left for the migration.

Some additions regarding pep8 rules: While pep8 helps a lot to gain readability the meaning of it is not to just follow the rules impassively, but to use pep8 to outline the meaning of a particular part of code. So my goal to apply pep8 rules was not to just match some kind of definition, but to understand your code by improving code readability while following pep8.

For example, pep8 allows multiple ways to split function calls. If there are function params containing function calls themselves, it is much more readable to split them one by line. For example, it is much more readable to write:

```
def foo(
    bar=1,
    baz=x(a, b, c),
    buz=y(d, e, f))
```

than to have something like this:

```
def foo(
    bar=1, baz=x(a, b, c),
    buz=y(d, e, f))
```

since it is harder to recognize that there are 3 params for foo() in the second way of coding style.

Nevertheless, it would be very helpful to reach a clean code base for me again by getting my pull request accepted, since there are some development branches in my repo for several new features and it is hard work to solve all merges and conflicts that are caused by a split brain.
